plugin/evm/vm_test.go

```diff
@@ -1749,12 +1752,14 @@ func TestBuildBlockWithInsufficientCapacity(t *testing.T) {
 		err error
 	)
 	for i := uint64(0); i < 2; i++ {
-		tx := types.NewContractCreation(
-			i,
-			big.NewInt(0),
-			acp176.MinMaxCapacity,
-			big.NewInt(ap0.MinGasPrice),
-			[]byte{0xfe}, // invalid opcode consumes all gas
+		tx := types.NewTx(
```

Comment:
Sorry, I didn't mean across the entire file, but we shouldn't be adding new code that relies on deprecated functions. I think you can leave the old usage as is (or change in another PR) but this is way too large a change to arbitrarily put into this PR.